File: src/main/java/tax/bilibili/nineblog/mapper/ArticleMapper.java
package tax.bilibili.nineblog.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import org.apache.ibatis.annotations.Mapper;
import tax.bilibili.nineblog.entity.Article;

/**
 * ArticleMapper
 *
 * @author sheip9
 * @since 2023/10/22 19:31
 */
@Mapper
public interface ArticleMapper extends BaseMapper<Article> {
}

File: server/src/main/java/tax/bilibili/nineblog/application/mapper/ArticleMapper.kt
package tax.bilibili.nineblog.application.mapper

import org.mapstruct.Mapper
import tax.bilibili.nineblog.application.entity.Article
import tax.bilibili.nineblog.application.model.dataTransfer.ArticleDTO
import tax.bilibili.nineblog.application.model.view.ArticleVo

@Mapper
interface ArticleMapper: BaseMapper<ArticleDTO, Article, ArticleVo>